File: metadata/e2etestsmainforked/pipelines/Workflow-Automated-GITHUB-FORK-1649079164232/code/src/main/scala/graph/Source_0.scala
package graph

import io.prophecy.libs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import config.ConfigStore._

object Source_0 {

  def apply(spark: SparkSession): DataFrame = {
    Config.fabricName match {
      case "dev" =>
        spark.read
          .format("json")
          .schema(
            StructType(
              Array(
                StructField("account_flags",     StringType, true),
                StructField("account_open_date", StringType, true),
                StructField("country_code",      StringType, true),
                StructField("customer_id",       StringType, true),
                StructField("email",             StringType, true),
                StructField("first_name",        StringType, true),
                StructField("last_name",         StringType, true),
                StructField("phone",             StringType, true)
              )
            )
          )
          .load("dbfs:/Prophecy/qa_data/json/CustomersDatasetInput.json")
          .cache()
      case _ =>
        throw new Exception("No valid dataset present to read fabric")
    }
  }

}

File: test/e2etestsmainforked2/pipelines/Workflow-Automated-GITHUB-FORK-1655429207129/code/job/graph/Source_0.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def Source_0(spark: SparkSession) -> DataFrame:
    return spark.read\
        .format("json")\
        .schema(
          StructType([
            StructField("account_flags", StringType(), True), StructField("account_open_date", StringType(), True), StructField("country_code", StringType(), True), StructField("customer_id", StringType(), True), StructField("email", StringType(), True), StructField("first_name", StringType(), True), StructField("last_name", StringType(), True), StructField("phone", StringType(), True)
        ])
        )\
        .load("dbfs:/Prophecy/qa_data/json/CustomersDatasetInput.json")
